radix-engine-interface/src/api/types/invocation/native.rs

```diff
@@ -304,6 +306,49 @@ pub enum WorktopMethod {
     Drain,
 }
 
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    EnumString,
+    EnumVariantNames,
+    IntoStaticStr,
+    AsRefStr,
+    Display,
+)]
+#[scrypto(TypeId, Encode, Decode, Describe)]
+#[strum(serialize_all = "snake_case")]
+pub enum ClockFunction {
+    Create,
+}
+
+#[derive(
+    Debug,
+    Clone,
+    Copy,
+    PartialEq,
+    Eq,
+    Hash,
+    PartialOrd,
+    Ord,
+    EnumString,
+    EnumVariantNames,
+    IntoStaticStr,
+    AsRefStr,
+    Display,
+)]
+#[scrypto(TypeId, Encode, Decode, Describe)]
+#[strum(serialize_all = "snake_case")]
+pub enum ClockMethod {
+    SetCurrentTime,
+    GetCurrentTimeToMinutePrecision,
```

Comment:
might be better to rename it to `GetCurrentTimeInMinutes`, "to minute precision" doesn't necessarily mean that the unit is also minutes